CoursesManager.UI/ViewModels/Courses/CourseDialogViewModel.cs

```diff
@@ -79,7 +82,11 @@ public CourseDialogViewModel(ICourseRepository courseRepository, IDialogService
                     };
 
             OriginalCourse = course;
-            Locations = new ObservableCollection<Location>(_locationRepository.GetAll());
+            var allLocations = _locationRepository.GetAll();
+            Locations = new ObservableCollection<Location>(allLocations);
```

Comment:
Kan opzich ook in 1 line maar dit is ook wel prima

Reply:
Toppp